src/main/java/entity/Event.java

```diff
@@ -0,0 +1,59 @@
+package entity;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by kowalskiy on 15.03.17.
+ */
```

Comment:
Two suggestions.
1. Participants must be List of Bands, not from Artist
2. address must be venue